src/PhpSpreadsheet/Worksheet/Worksheet.php

```diff
@@ -1191,8 +1191,11 @@ public function setCellValueExplicitByColumnAndRow($columnIndex, $row, $value, $
      */
     public function getCell($pCoordinate, $createIfNotExists = true)
     {
+        // Uppercase coordinate
+        $pCoordinateUpper = strtoupper($pCoordinate);
+
         // Check cell collection
-        if ($this->cellCollection->has(strtoupper($pCoordinate))) {
+        if ($this->cellCollection->has($pCoordinateUpper)) {
```

Comment:
This seems like a bug to me.  Shouldn't `get` use the same key we checked with `has`?  Based on the code in `Cells` I think this can return null.